docs/scenarios.md

```diff
@@ -1,36 +1,34 @@
 ---
-title: Scenarios
+title: Koperator capablities
 weight: 400
 ---
 
+As highlighted in the [features section]({{< relref "../_index.md#features" >}}), {{< kafka-operator >}} removed the reliance on StatefulSet,and supports several different usecases.
 
-
-As highlighted in the [features section]({{< relref "../_index.md#features" >}}), we removed the reliance on StatefulSet, we support several different scenarios.
-
-> Note: this is not a complete list, if you have a specific requirement or question, [contact us](mailto:calisti-support@cisco.com).
+> Note: This is not a complete list, if you have a specific requirement or question, see our [support]({{< relref "/sdm/koperator/support.md">}}) options.
 
 ## Vertical capacity scaling
 
-We've encountered many situations in which the horizontal scaling of a cluster is impossible. When **only one Broker is throttling** and needs more CPU or requires additional disks (because it handles the most partitions), a StatefulSet-based solution is useless, since it does not distinguish between replicas' specifications. The handling of such a case requires *unique* Broker configurations. If we need to add a new disk to a unique Broker, we waste a lot of disk space (and money) with a StatefulSet-based solution, since it can't add a disk to a specific Broker, the StatefulSet adds one to each replica.
+You may have encountered situations where the horizontal scaling of a cluster is impossible. When **only one Broker is throttling** and needs more CPU or requires additional disks (because it handles the most partitions), a StatefulSet-based solution is useless, since it does not distinguish between replicas' specifications. The handling of such a case requires *unique* Broker configurations. If there is a need to add a new disk to a unique Broker, there can be waste of disk space (and money) with a StatefulSet-based solution, since it can't add a disk to a specific Broker, the StatefulSet adds one to each replica.
```

Comment:
**Optional**: We **MAY** consider the following phrasing.

```suggestion
You may have encountered situations where the horizontal scaling of a cluster is impossible. When **only one Broker is throttling** and needs more CPU or requires additional disks (because it handles the most partitions), a StatefulSet-based solution is useless, since it does not distinguish between replicas' specifications. The handling of such a case requires *unique* Broker configurations. If there is a need to add a new disk to a unique Broker, there can be a waste of disk space (and money) with a StatefulSet-based solution, since it can't add a disk to a specific Broker, the StatefulSet adds one to each replica.
```